pages/features.html

```diff
@@ -6,84 +6,112 @@
+  <h1>{% trans "Everything You Need, However You DJ" %}</h1>
 
   <div style="clear: both;"></div>
   <div class="halfbox_left">
-    <H2>{% trans "Four Decks, Advanced Controls" %}</H2>
+    <h2>{% trans "Four Decks" %}</h2>
     <p>
-      {% trans "Drop a song onto a deck in Mixxx to kick off your next mix. Each deck features a scratchable, scrolling waveform that marks beats and cue points, along with a whole-song waveform overview for quick seeking." %}
+      {% trans "Drop a song onto a deck in Mixxx to start playing right away. Keep your eye on beats and cue points with a scratchable, scrolling waveform together with a whole-song overview waveform for quick seeking." %}
     </p>
 
-    <p><b>{% trans "Time Stretch and Vinyl Emulation" %}</b><br>
-      {% trans "Speed up and slow down songs without changing their pitch with time stretching, or just change the musical pitch so your songs play in harmony. Our high-quality interpolator can also reproduce classic vinyl sounds like backspins." %}
+    <p><b>{% trans "Pitch and Key Control" %}</b><br>
+      {% trans "Change the tempo of songs without changing their pitch with keylock, or change the pitch without breaking sync so your songs play in harmony. Temporarily nudge the tempo faster or slower for manual beatmatching, with smooth changes in pitch just like vinyl." %}
     </p>
 
     <p><b>{% trans "Beat Looping" %}</b><br>
-      {% trans "Found a sweet loop or need to extend your mix while you prep the next track? Instantly loop a 4, 8, or 16 beat segment with the press of a button." %}
+      {% trans "Need to extend your mix while you bring in the next track? Instantly loop a 4, 8, or 16 beat segment with ease." %}
     </p>
 
-    <p><b>{% trans "Powerful Master Sync" %}</b><br>
+    <p><b>{% trans "Master Sync" %}</b><br>
       {% trans "Engage master sync on your decks and they'll stay locked in time even if you change speed.  Construct remixes on the fly with multiple tracks and loops without losing control." %}
     </p>
 
     <p><b>{% trans "Hotcues" %}</b><br>
-      {% trans "Four hotcue points can be dropped or triggered with ease. Our playback engine is finely tuned for accurate, rapid fire hotcue triggering so you can mash and remix as hard as you want to without a hiccup." %}
+      {% trans "Set hotcues to mark places in tracks. Remix and mash up tracks with accurate, rapid-fire hotcue triggering." %}
     </p>
   </div>
-
   <div class="halfbox_right">
     <img width="362px" src="{% static '/static/images/feature_waveform-2.0.png' %}">
     <p><b>{% trans "Beat Rolls and Censor" %}</b><br>
-      {% trans "Play with rhythm by triggering short loops and reverse-playback effects.  The track stays in time so you don't miss a beat." %}
+      {% trans "Play with rhythm by triggering short loops and reverse-playback effects. The track stays in time so you don't miss a beat." %}<!--TODO: Write a better way of explaining slip mode.-->
```

Comment:
nit: best to use jinja comments instead of HTML so that it doesn't end up in the final HTML, e.g. 

`{# comment #}`